articles/azure/azs-how-ps-update-azure-stack.md

```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
 > [!CAUTION]
 > In the 1802 example you can see that you need to install a Hotfix before you apply the upgrade. That is one of the many reasons why you should always check the official documentation.
 
-# High Level Overview of the process
+## High Level Overview of the process
 
 > [!IMPORTANT]
 >
->
 > We strongly recommend that you notify users of any maintenance operations, and that you schedule normal maintenance windows during non-business hours as much as possible. Maintenance operations may affect both user workloads and portal operations.
 
 1. Raise a pre-emptive case with Microsoft prior to starting the Update.
 
-2. Download required packages.
-* Full Manual procedure can be found here -> [more](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-apply-updates#import-and-install-updates)
```

Comment:
You can find the full manual procedure [here](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-apply-updates#import-and-install-updates).